src/ui/localization/locale/en.json

```diff
@@ -54,6 +54,7 @@
   "error_page.error_message_already_subscribed": "You are already subscribed to this product.",
   "error_page.error_message_already_purchased": "You have already purchased this product.",
   "error_page.error_message_missing_email_error": "Email is required to complete the purchase.",
```

Comment:
I guess this one has been replaced by a Stripe error message, right?

Reply:
Yep, there's still a bit of a mess with error codes for the public SDK getting tangled with the error codes needed by the UI but I decided to not fix it here.